xml/depl_nodes.xml

```diff
@@ -4136,9 +4136,7 @@ getent group postgres | cut -d: -f3</screen>
     Making &o_meter; highly available requires no special
     configuration&mdash;it is sufficient to deploy the roles
     <guimenu>ceilometer-server</guimenu> and
-    <guimenu>ceilometer-polling</guimenu> on a cluster. The cluster needs to
-    consist of an odd number of nodes, otherwise the &o_meter; deployment will
-    fail.
+    <guimenu>ceilometer-polling</guimenu> on a cluster. If you are using MongoDB for your backend the cluster must have a minimum of three nodes. If you are using MySQL or PostgreSQL, you may use two nodes. (MongoDB is the recommended backend for best performance.)
```

Comment:
And could you please mention this change in the docupdates, too? (see the empty skeleton in depl_docupdates.xml, sec.deploy.docupdates.c7_maint2) Thanks.